scripts/planktoscope/segmenter/__init__.py

```diff
@@ -813,11 +813,16 @@ def segment_path(self, path, ecotaxa_export):
             )
         )
 
-        project = self.__global_metadata["sample_project"].replace(" ", "_")
-        sample = self.__global_metadata["sample_id"].replace(" ", "_")
-        date = datetime.datetime.utcnow().isoformat()
+        def clean_name(name):
+            s1="".join(c for c in name if c.isalnum() or c == " ")
+            s1 = s1.replace(" ","_")
```

Comment:
I would be nice to insert here something that will remove the double underscore that could be introduced:
```
while "__" in name:
    name = name.replace("__", "_")
```

Reply:
dacodac